config/samples/layout/base/glance_v1beta1_glance.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: glance
 spec:
   serviceUser: glance
+  rabbitmqClusterName: rabbitmq
```

Comment:
maybe I can remove this and check that we have no regressions with the current kuttl tests when this parameter is not passed.